src/github.ts

```diff
@@ -226,30 +226,39 @@ export class GitHub {
-    //  workflow: https://github.com/docker/actions-toolkit/actions/runs/9552208295
-    // https://github.com/docker/actions-toolkit/issues/367
-    const artifactRelativeURL = `./${GitHub.runId}/${opts.uploadRes.url.split('/').slice(-2).join('/')}`;
-
-    // prettier-ignore
-    const sum = core.summary
-      .addHeading('Docker Build summary', 2)
-      .addRaw(`<p>`)
+    const sum = core.summary.addHeading('Docker Build summary', 2);
+
+    if (opts.uploadRes) {
+      // we just need the last two parts of the URL as they are always relative
+      // to the workflow run URL otherwise URL could be broken if GitHub
+      // repository name is part of a secret value used in the workflow. e.g.:
+      //  artifact: https://github.com/docker/actions-toolkit/actions/runs/9552208295/artifacts/1609622746
+      //  workflow: https://github.com/docker/actions-toolkit/actions/runs/9552208295
+      // https://github.com/docker/actions-toolkit/issues/367
+      const artifactRelativeURL = `./${GitHub.runId}/${opts.uploadRes.url.split('/').slice(-2).join('/')}`;
+
+      // prettier-ignore
+      sum.addRaw(`<p>`)
         .addRaw(`For a detailed look at the build, download the following build record archive and import it into Docker Desktop's Builds view. `)
         .addBreak()
         .addRaw(`Build records include details such as timing, dependencies, results, logs, traces, and other information about a build. `)
         .addRaw(addLink('Learn more', 'https://www.docker.com/blog/new-beta-feature-deep-dive-into-github-actions-docker-builds-with-docker-desktop/?utm_source=github&utm_medium=actions'))
       .addRaw('</p>')
       .addRaw(`<p>`)
         .addRaw(`:arrow_down: ${addLink(`<strong>${Util.stringToUnicodeEntities(opts.uploadRes.filename)}</strong>`, artifactRelativeURL)} (${Util.formatFileSize(opts.uploadRes.size)} - includes <strong>${refsSize} build record${refsSize > 1 ? 's' : ''}</strong>)`)
-      .addRaw(`</p>`)
-      .addRaw(`<p>`)
-        .addRaw(`Find this useful? `)
-        .addRaw(addLink('Let us know', 'https://docs.docker.com/feedback/gha-build-summary'))
-      .addRaw('</p>');
+      .addRaw(`</p>`);
+    } else {
+      // prettier-ignore
+      sum.addRaw(`<p>`)
+        .addRaw(`The table below provides a small summary of your build.`)
+        .addBreak()
+        .addRaw(`For a detailed look at the build, including timing, dependencies, results, logs, traces, and other information, consider enabling the export of the build record so you can import it into Docker Desktop's Builds view. `)
+        .addRaw(addLink('Learn more', 'https://www.docker.com/blog/new-beta-feature-deep-dive-into-github-actions-docker-builds-with-docker-desktop/?utm_source=github&utm_medium=actions'))
+      .addRaw(`</p>`);
+    }
+
+    // Feedback survey
+    sum.addRaw(`<p>`).addRaw(`Find this useful? `).addRaw(addLink('Let us know', 'https://docs.docker.com/feedback/gha-build-summary')).addRaw('</p>');
```

Comment:
What if we moved this to the end? The inputs and bake def are collapsed by default, so it should still be visible. There's something odd about having this question appear right before the summary.

Reply:
Agreed but needs to consider the table can be quite huge if building many tarets with bake like

![image](https://github.com/docker/actions-toolkit/assets/1951866/d6e43879-3f89-40d2-bde4-bd0f70b91caa)

Reply:
Yes, but this one is a bit of a special case, right? Do people tend to build more than ~5 or so targets in a run?

Reply:
@colinhemmings Agreed, maybe right after the table then? We can look at this as follow-up as well.

Reply:
if they expand the sections and look at the summary, would they scroll back up later to give feedback?